src/main/java/pl/wojciechkostecki/revel/model/Menu.java

```diff
@@ -0,0 +1,26 @@
+package pl.wojciechkostecki.revel.model;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.util.HashSet;
+import java.util.Set;
+
+@Entity
+@Table(name = "menu")
```

Comment:
Why in Local entity you have used `locals` as table name and here you are using `menu`. Stay consistent, either use plural or singular, don't mix them. It stinks.